src/index.html

```diff
@@ -12,11 +12,45 @@
     <title>Aurator</title>
 </head>
 <body class="app">
-    <img src="img/desktop/naprawa__foto.png" alt="foto">
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-12">
-                <h2>Aurator</h2>
+    <div class="container-fluid">
+        <div class="top">
+            <img class="" src="../img/desktop/banner__bg.png">
+            <div class="nav">
+                <div class="nav-logo">
+                    <img class="d-none d-sm-block" src="../img/desktop/nav__logo.png">
+                    <img class="d-block d-sm-none" src="../img/mobile/nav__logo.png">
+                </div>
```

Comment:
Differences between `img/desktop/nav__logo.png` for mobile and desktop are to small to have to two imgs.